### DISCUSSION TITLE
How do I install Unit test module?

### DISCUSSION THREAD

**Discussion by tomwjerry** [Codeception/Codeception#6612] [Q&A] [answered]
As for the title, how do I install the Unit test module on Codeception 5?
I have tried
php vendor/bin/codecept bootstrap
php vendor/bin/codecept init Unit

But when I run php vendor/bin/codecept generate:test unit Test I got the error "Suite unit was not loaded". Did I miss some steps?

**Comment by Naktibalda** [comment, marked as answer]
Suite names are case-sensitive - `php vendor/bin/codecept generate:test Unit Test`
